audit/README.md

```diff
@@ -59,7 +59,8 @@ future changes by simply running `terraform apply
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.12 |
+| terraform | ~> 0.12.0 |
+| aws | ~> 2.0 |
 
 ## Providers ##
 
```

Comment:
How are there no providers here?

Reply:
That is strange- I was relying on the output from `terraform-docs`.  I added in the default `aws` providers in https://github.com/cisagov/cool-accounts/pull/61/commits/14361cc966bf23fbe365537323e4ee5fe9b5fedc.